packages/core/src/server/middlewares/server-side-rendering.ts

```diff
@@ -67,6 +84,11 @@ export const serverSideRendering = async (
     // https://github.com/smooth-code/loadable-components/pull/239#issuecomment-482501467
     const customExtractor = extractor as Extractor;
 
+    // Run afterSSR actions. It runs at this point because we want to run it
+    // before taking the state snapshot. This gives the user a chance to
+    // modify the state before sending it to the client
+    runAfterSSRActions(store, ctx);
```

Comment:
Do we have to `await` here? 🤔  I think so? Same thing in the other case.
```suggestion
    await runAfterSSRActions(store, ctx);
```

Reply:
Oh, you're right! For some reason I thought it should not be necessary to await here if I await **inside** each of the `afterSSR()` calls but that's not the case of course.